### PR TITLE
more documentation around creating preview sites

### PR DESCRIPTION
**Pull request by mstbbs**
Just some more context around how things work. This could all be built out much more, but I have been focusing on not dropping any balls on this migration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,20 @@ coming soon
 
 # Working on Docs
 
+## Datadog Staff
+
+* Always branch off of master
+* Create a branch `slack_handle/your_feature` if you would like to create a preview site and run tests
```
Comment (technovangelist) [inline README.md]: i would drop the 'if you would like....' - always create the branch. no ifs

File: README.md [hunk 2]
```diff
@@ -63,6 +63,20 @@ coming soon
...
+* Never commit directly to master
```
Comment (kmshultz) [inline README.md]: This seems redundant with the first item. Maybe combine them both: "Always branch off of master; never commit directly to master"

File: README.md (hunk 1 above)
Comment (kmshultz) [inline README.md]: I don't think the 'if' is meant to operate on 'Create'; it operates on 'you would like to create a preview site" 

Probably 'Name' is better: "**Name** your branch `slack_handle/your_feature` if you would like to..."

Comment (technovangelist) [inline README.md]: but i think its a good naming strategy and we should encourage it. It seems to be what most do anyway

File: README.md (hunk 2 above)
Comment (technovangelist) [inline README.md]: yes

**Pull request merged**